doc/src/EntitiesTraitsSpecifications.dox

```diff
@@ -78,30 +79,38 @@
  * to ensure data is moved to long-term storage.
  *
  * The manager doesn't need to check an extensive list of which entity
- * types are represented by data on disk - it just checks if any given
- * entity has that specific trait.
+ * types may be registered with data on disk - it just checks if any
+ * given entity has that specific trait.
  *
  * @section entity_data Entity Data
  *
- * Every trait associated with an entity can define zero or more
+ * Every trait associated with an @ref entity can define zero or more
  * key/value pairs of data (properties) that the entity (may) hold.
  *
- * This is the mechanism by which an @ref entity_reference is converted
- * into the data a @ref host requires at runtime - the host asks the
- * @ref manager to @ref resolve the data for one or more specific
- * traits.
+ * Traits are the mechanism by which a @ref host requests specific data
+ * about an entity at runtime. The @ref openassetio.hostAPI.Manager.Manager.resolve
+ * "resolve" method asks the manager to retrieve the data for a specific
+ * set of traits for one or more @ref entity_reference "entity
+ * references".
  *
  * Continuing the example above, to determine the path to the character
  * model, the host would resolve the data for the `locatableContent`
- * trait, and read the `url` property to locate the data.
+ * trait, and read the `location` property to locate the data.
  *
- * Within a tool that displays EXR image sequences, the
- * `locatableContent`, `image` `frameRange`, and `playback` traits could
- * be queried. The data provided by the manager is then used to control
- * the colorspace and frame range/rate used for playback. This can avoid
+ * A tool that displays EXR image sequences may resolve the
+ * `locatableContent`, `image` `frameRange`, and `playback` traits. The
+ * data provided by the manager then being used to control the
+ * colorspace and frame range/rate used for playback. This avoids
  * fragile mechanisms such as extension-based mappings and brute-force
  * filesystem searches.
  *
+ * The @ref Context passed to the API methods by the host plays an
+ * important role during resolution. Its @ref locale property also makes
+ * use of traits to specify details of the host environment. For
+ * example, the `usesUrls` trait can be set in the locale, to inform the
```

Comment:
[fix] Drop this for now as were not certain about this behaviour, as it prohibits gender store/recall of entities.

Reply:
Agreed, in 2022 we should absolutely permit entities to store and recall their gender.

Not sure where we land on resolving self-made comments?  I'm happy for this to be resolved.